nautobot/extras/models/customfields.py

```diff
@@ -31,6 +31,8 @@
     StaticSelect2,
     StaticSelect2Multiple,
     add_blank_choice,
+    CommentField,
+    SmallTextarea,
 )
```

Comment:
Nitpick: let's keep the entries here in alphabetical order rather than adding to the end.